src/files-card-body.scss

```diff
@@ -194,6 +194,11 @@
         th, td {
             padding: var(--pf-v5-global--spacer--sm);
             border-block-start: none;
+
+            /* Do not visually highlight files when double clicking */
+            a {
+              user-select: none;
```

Comment:
I'm not completely sure this is what we should do as this also block copying filename (or at least won't show the highlight overlay when you manually try to select the text by holding down the left mouse button). I think you should only block the default browser behavior when clicking on the file by which it also selects the text.

Reply:
Good point, I found an alternative way to do it https://developer.mozilla.org/en-US/docs/Web/CSS/::selection

That does make it a bit confusing when stuff is "selected"